doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -1279,7 +1292,255 @@ Default settings for logging are the following:
 - logging level: INFO;
 - rolling policy: whenever file size reaches 100 MB.
 
-# 17. Migrating to Remote Database Host
+# 17 Management REST API
+
+Central server has a REST API that can be used to do all the same server configuration operations that can be done
+using the web UI.
+
+Management REST API is protected with an API key based authentication. To execute REST calls, API keys need to be created.
+
+REST API is protected by TLS. Since server uses self-signed certificate, the caller needs to accept this (for example
+with `curl` you might use `--insecure` or `-k` option).
+
+Requests sent to REST API have a *limit for maximum size*. If a too large request is sent
+to REST API, it will not be processed, and http status 413 Payload too large will be returned.
+There is a different limit for binary file uploads, and for other requests.
+
+Limits are
+- 10MB for file uploads
+- 50KB for other requests
+
+REST API is also *rate limited*. Rate limits apply per each calling IP. If the number of calls
+from one IP address exceeds the limit, endpoints return http status 429 Too Many Requests.
+
+Limits are
+- 600 requests per minute
+- 20 requests per second
+
+If the default limits are too restricting (or too loose), they can be overridden with command line arguments. Limits are set with
+application properties
+- `request.sizelimit.regular`
+- `request.sizelimit.binary.upload`
+- `ratelimit.requests.per.second`
+- `ratelimit.requests.per.minute`
+
+**Note:** These properties have been deprecated since 7.3.0, please use `request-sizelimit-*` & `rate-limit-requests-per-*` [proxy-ui-api parameters](ug-syspar_x-road_v6_system_parameters.md#39-management-rest-api-parameters-proxy-ui-api) instead
+
+Size limit parameters support formats from Formats from [DataSize](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/util/unit/DataSize.html),
+for example `5MB`.
+
+New command line arguments can be added, not replaced, using the configuration file `local.properties`.
+Example of `/etc/xroad/services/local.properties` with modifications:
+
+```properties
+XROAD_PROXY_UI_API_PARAMS=-Dratelimit.requests.per.second=100 -Drequest.sizelimit.binary.upload=1MB
+```
+
+## 17.1 API key management operations
```

Comment:
Please review all the request examples and check that they use [Central Server roles](https://github.com/nordic-institute/X-Road/blob/release-7.3/doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md#21-user-roles) and Central Server API endpoints (including example request/response payloads).